File: src/main/java/me/viniciusarnhold/altaria/utils/Logs.java
package me.viniciusarnhold.altaria.utils;

import me.viniciusarnhold.altaria.core.App;
import org.apache.logging.log4j.Level;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

/**
 * Created by Vinicius.
 *
 * @since ${PROJECT_VERSION}
 */
public class Logs {

    private static final Logger logger = LogManager.getLogger(App.class);


    public static void forThrowable(Throwable t) {
        logger.error(t);
    }

    public static void forMessage(Level level, String message) {
        logger.log(level, message);
    }

    public static void forMessage(String message) {
        logger.log(Level.ERROR, message);
    }

}

File: core/src/main/kotlin/me/viniciusarnhold/altaria/utils/Logs.kt
package me.viniciusarnhold.altaria.utils

import me.viniciusarnhold.altaria.core.App
import org.apache.logging.log4j.Level
import org.apache.logging.log4j.LogManager

/**
 * Created by Vinicius.

 * @since 1.0
 */
object Logs {

    private val logger = LogManager.getLogger(App::class.java)

    fun forThrowable(t: Throwable) {
        logger.error(t)
    }

    fun forMessage(level: Level, message: String) {
        logger.log(level, message)
    }

    fun forMessage(message: String) {
        logger.log(Level.ERROR, message)
    }

}
